build.gradle.kts

```diff
@@ -27,17 +27,18 @@ repositories {
     mavenCentral()
     maven { url = uri("https://repo.papermc.io/repository/maven-public/") }
     maven { url = uri("https://repo.codemc.io/repository/maven-public/") }
-    maven { url = uri("https://repo.dmulloy2.net/nexus/repository/public/") }
     maven { url = uri("https://maven.enginehub.org/repo/") }
+    maven { url = uri("https://jitpack.io") }
```

Comment:
Could you add a dependency resolution strategy for decent holograms only, please?

Given everyone can trigger builds on jitpack, pulling other dependencies, but DH can cause unwanted side effects aka broken jars. I'd recommend attempting pulling DH only from Jitpack.